Update to Infinite cage S120 (#1244)
pi-base/data: spaces/S000120/properties/P000089.md

---
space: S000120
property: P000089
value: false
---

A continuous map $f:X\to X$ without a fixed point can be defined by setting
$f:D_n\ni (x,y,z)\mapsto (\frac{n}{n+1}x,y+2,\frac{n^2}{(n+1)^2}z)\in
D_{n+1}$,
where $n\geq 1$ and $D_n=A_n\cup B_n\cup C_n$.

> Consider a convergent sequence $p_n\to p$ in $X$.
> If $p\neq(0,y,0)$ for $y\in \mathbb R$, then $p_n$ is eventually contained in $D_n$ for $D_n\ni p$, while $f$ is continuous when restricted to $D_n$.
> If $p=(0,y,0)$, then the first and the thrid coordinate of $p_n$ tend to zero and the same is true for $f(p_n)$. Convergence on the second coordinate is evident.

Reviewer: ```suggestion
To show the continuity of $f$, consider a convergent sequence $p_n\to p$ in $X$.
If $p\neq(0,y,0)$ for $y\in \mathbb R$, then $p_n$ is eventually contained in $D_n$ for $D_n\ni p$, and $f$ is continuous when restricted to $D_n$.
If $p=(0,y,0)$, then the first and the third coordinates of $p_n$ tend to zero and the same is true for $f(p_n)$. Convergence in the second coordinate is evident.
```
